### PR TITLE
Governance CLI changes

### LINKED ISSUE #3293: Governance CLI command to show account information
### Expected Behavior

`governance:show --account` shows information on what proposals the account has voted on, how many pending withdrawals they have.

### Current Behavior

No convenient way to view any of this information

### LINKED ISSUE #3294: Governance CLI command to withdraw deposits
### Expected Behavior

`governance:withdraw` allows users to withdraw proposal deposits 

### Current Behavior

No convenient way to do this

### PR DESCRIPTION
**Pull request by asaj**
## Description

This PR adds `governance:withdraw` for withdrawing proposal deposits, combines `governance:view` and `governance:viewhotfix` into `governance:show` for consistency with other modules, and adds `governance:show --account`

## Other changes

Add test for `governance:vote`
## Tested

Unit tests
## Related issues

- Fixes #3293 
- Fixes #3294 

## Backwards compatibility

CLI commands changed

### CODE REVIEW

File: packages/cli/src/commands/governance/show.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    account: flags.string({
+      exclusive: ['proposalID', 'hotfix'],
+      description: 'Address of account or voter',
+    }),
+    hotfix: flags.string({
+      exclusive: ['account', 'proposalID'],
+      description: 'Hash of hotfix proposal',
+    }),
+    notwhitelisted: flags.boolean({
+      description: 'List validators who have not whitelisted the speicified hotfix',
```
Comment (mrsmkl) [inline packages/cli/src/commands/governance/show.ts]:
```suggestion
      description: 'List validators who have not whitelisted the specified hotfix',
```

File: packages/cli/src/commands/governance/withdraw.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({ required: true, description: "Proposer's address" }),
+  }
+
+  static examples = ['withdraw --from 0x5409ed021d9299bf6814279a6a1411a7e866a631']
+
+  async run() {
+    const res = this.parse(Withdraw)
+    console.log(res.flags.from)
```
Comment (mrsmkl) [inline packages/cli/src/commands/governance/withdraw.ts]:
```suggestion
```

File: packages/contractkit/src/wrappers/Governance.ts [hunk 3]
```diff
@@ -381,13 +404,44 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
...
+  }
+
   /**
    * Returns whether a given proposal is queued.
    * @param proposalID Governance proposal UUID
    */
   isQueued = proxyCall(this.contract.methods.isQueued, tupleParser(valueToString))
 
-  /**
+  /*
```
Comment (mrsmkl) [inline packages/contractkit/src/wrappers/Governance.ts]: Looks like ** is needed, otherwise it doesn't go to  docs.

**Review by mrsmkl — APPROVED** [review]
Great! Didn't test `withdraw`, but `show` seems to work.

**Review by aslawson — COMMENTED** [review]
LGTM
tested the `show` command with regular proposal (non-hotfix) and the `withdraw` command

**Review by aslawson — APPROVED** [review]

File: packages/contractkit/src/wrappers/Governance.ts (hunk 3 above)
Comment (asaj) [reply packages/contractkit/src/wrappers/Governance.ts]: Oops!

**Pull request merged**